README.md

```diff
@@ -66,34 +66,24 @@ For more examples on how to use the client, refer to the [examples](https://gith
 
 ## Middleware
 
-In addition to the API operations, the SDK also provides a middleware that can be used to inject the active session into the request's context.
-The Clerk middleware expects a `clerk.Client` and resolves the active session using the incoming session cookie.
+The SDK provides a middleware, [`WithSessionV2`](https://pkg.go.dev/github.com/clerkinc/clerk-sdk-go/v2/clerk#WithSessionV2), that injects the active session into the request's context.
 
-The active session object will be added in the request's context using the key `clerk.ActiveSession`.
+The active session's claims can then be fetched using [`SessionFromContext`](https://pkg.go.dev/github.com/clerkinc/clerk-sdk-go/v2/clerk#SessionFromContext).
```

Comment:
❓ `fetched` may raise a question whether there's an HTTP request involved here? Not sure.

```suggestion
The active session's claims can then be accessed using [`SessionFromContext`](https://pkg.go.dev/github.com/clerkinc/clerk-sdk-go/v2/clerk#SessionFromContext).
```

Reply:
Very good point! Changed.